cypress/integration/rules_spec.js

```diff
@@ -111,6 +111,40 @@ describe('Rules page', () => {
       cy.task('resetState');
     });
 
+    it('copying a rule should add it to the list', () => {
+      cy.visit('/rules');
+      cy.contains('.table .tbody .tr a', testRuleName)
+        .and('have.attr', 'href', `/rules/rule/${testRuleName}`)
+        .click();
+
+      cy.contains('.heading--large', testRuleName);
+      cy.contains('.button--small', 'Copy').click();
+      cy.contains('.heading--large', 'Copy a rule');
+
+      const newName = 'testRule2';
+      cy.editJsonTextarea({ data: { name: newName }, update: true });
```

Comment:
This test is occasionally failing. What I've found is that sometimes the rerender with the default content has not happened before this editJsonTestArea call. Adding in a `contains` command for the fields that should be present seems to fix it because of cypress' built in retry functionality. 
```suggestion
      cy.contains('.ace_string', testRuleName);
      cy.editJsonTextarea({ data: { name: newName }, update: true });
```